src/controllers/book.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { Book } = require('../models');
+const { 
+  createItem, 
+  getAllItems, 
+  getItemById, 
+  updateItemById,
+  deleteItemById,
+  } = require('./helpers');
```

Comment:
The other files in this directory use 4 spaces of indentation but only 2 are here. Also the closing bracket on line 9 should be at the same level as the start of the text on the opening line of the function:
```suggestion
    createItem, 
    getAllItems, 
    getItemById, 
    updateItemById,
    deleteItemById,
} = require('./helpers');
```